src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -528,5 +543,9 @@ public static String removeEnclosingSquareBrackets(String str) {
         }
         
         return str.substring(1, str.length() - 1);
+>>>>>>> master
+=======
+        return stringsAfterTrim;
+>>>>>>> c13ef4e3cda383892bd796acf8a9cce272709070
```

Comment:
These are the signs of merge conflicts. You should not commit those; rather, you should fix those (see which lines need to be there and which ones don't). If you have not encountered this kind of stuff before, have a look at [this doc](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#Basic-Merge-Conflicts).